### PR TITLE
Added blog post for grants launch

### PR DESCRIPTION
**Pull request by wonderwomancode**
All in the title

### CODE REVIEW

File: unlock-protocol-com/blog/grants-program-updates.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Unlock Announces Grants Program Upgrade
+subTitle: Improvements go live to streamline processes and accelerate growth for the Unlock ecosystem
+description: Improvements go live to streamline processes and accelerate growth for the Unlock ecosystem.
+author: Angela Steffens
+date: November, 1 2022
```
Comment (ccarfi) [inline unlock-protocol-com/blog/grants-program-updates.md:6]: @wonderwomancode date is in incorrect format (the comma is in the wrong place) so post does not render on the list of posts at ./blog/

Comment (wonderwomancode) [reply unlock-protocol-com/blog/grants-program-updates.md]: Fixed it

File: unlock-protocol-com/blog/grants-program-updates.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+image: /images/blog/grants-upgrade-share.png
+---
+
+## Improvements  go live to streamline processes and accelerate growth for the Unlock ecosystem.
```
Comment (ccarfi) [inline unlock-protocol-com/blog/grants-program-updates.md:10]: line 10 is not needed, redundant with the description / subTitle above. please remove.

Comment (wonderwomancode) [reply unlock-protocol-com/blog/grants-program-updates.md]: Removed

File: unlock-protocol-com/blog/grants-program-updates.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+- [ETHAnglia](https://ethanglia.org/)
+
+**Education**
+
+- [Web3Academy](https://w3academy.io/)
+- [dProgramming University](https://dprogramminguniversity.com/courses/unlock-protocol-101/)
+- [Unlock Showcase](https://www.unlockshowcase.com/)
+
+With the upgraded grants program, we can’t wait to see what the community creates next! You can fill out a grant application here!
```
Comment (ccarfi) [inline unlock-protocol-com/blog/grants-program-updates.md:75]: `fill out a grant application here` should link to https://docs.unlock-protocol.com/governance/grants-bounties/udt-grantee-handbook/#submitting-an-application

**Review by ccarfi — APPROVED** [review]

**Pull request merged**